Rubberduck.VBEditor.VBA/SafeComWrappers/VB/CodePane.cs

```diff
@@ -102,17 +102,22 @@ private void ForceFocus()
 
             Show();
 
-            var window = VBE.MainWindow;
-            var mainWindowHandle = window.Handle();
-            var handle = window.Handle().FindChildWindow(Window.Caption);
-            
-            if (handle != IntPtr.Zero)
+            using (var vbe = VBE)
+            using (var mainWindow = vbe.MainWindow)
+            using (var window = Window)
             {
-                NativeMethods.ActivateWindow(handle, mainWindowHandle);
-            }
-            else
-            {
-                System.Diagnostics.Debug.WriteLine("CodePane.ForceFocus() failed to get a handle on the MainWindow.");
+                var mainWindowHandle = mainWindow.Handle();
+                var handle = mainWindow.Handle().FindChildWindow(window.Caption);
+
+                if (handle != IntPtr.Zero)
+                {
+                    NativeMethods.ActivateWindow(handle, mainWindowHandle);
+                }
+                else
+                {
+                    System.Diagnostics.Debug.WriteLine(
+                        "CodePane.ForceFocus() failed to get a handle on the MainWindow.");
```

Comment:
we don't have a Logger available here?

Reply:
Hmm. I missed that (wasn't really focusing on that part. Will check and make sure to use Nlog there.